src/components/atoms/bar-chart-content.tsx

```diff
@@ -102,6 +104,28 @@ export const BarChartContent: FC<BarChartContentProps> = ({
     <TrendLine lineWidth={3} color={primaryColor} {...props} />
   );
 
+  const Label: FC<{
+    data: any;
+    y: (value: number) => void;
+    x: () => void;
+    bandwidth: number;
+  }> = (props) => {
+    const {x, y, bandwidth, data} = props;
+    return data.map((value: {value: number}, index: number) =>
+      index === data.length - 1 ? (
+        <View
+          accessible={true}
+          style={[styles.label, {top: y(value.value) - 35}]}>
+          <View style={styles.triangle} />
+          <View style={styles.triangle2} />
+          <Text maxFontSizeMultiplier={1} style={{textAlign: 'center'}}>
```

Comment:
All text should have one of the `text.` styles from the theme so that fonts etc are applied; and such static styles should go in the stylesheet like `{ ...text.someStyle, textAlign: 'center' }` (you should get warnings for static styles like this if you have an eslint plugin set up in your editor)